lib/Dialect/FIRRTL/Transforms/GrandCentral.cpp

```diff
@@ -446,6 +450,37 @@ void GrandCentralPass::runOnOperation() {
       if (visitor.hasFailed())
         return signalPassFailure();
       AnnotationSet annotations(&op);
+      // Insert an instantiated interface
+      if (auto viewAnnotation = annotations.getAnnotation(
+              "sifive.enterprise.grandcentral.GrandCentralView$"
+              "SerializedViewAnnotation")) {
+        auto tpe = viewAnnotation.getAs<StringAttr>("type");
+        if (tpe && tpe.getValue() == "parent") {
+          auto name = viewAnnotation.getAs<StringAttr>("name");
+          auto defName = viewAnnotation.getAs<StringAttr>("defName");
+          auto guard = OpBuilder::InsertionGuard(builder);
+          builder.setInsertionPointToEnd(cast<FModuleOp>(op).getBodyBlock());
+          auto instance = builder.create<sv::InterfaceInstanceOp>(
+              circuitOp->getLoc(),
+              interfaces.lookup(defName.getValue()).getInterfaceType(), name,
+              builder.getStringAttr("__" + defName.getValue() + "__"));
```

Comment:
Does this name need to get uniquified in some way? What happens if there's already something called "__foo__" around, and I'm adding this GCT view "foo"?

Reply:
Let me check. I think this is the same possibly unsafe approach being used for lowering instances to binds in `LowerToHW` and generating wire/reg symbols.  Let me see what happens if there's a conflict.

Reply:
Yeah, this is totally unsafe. 😅  I can update this to actually generate a safe name. I was cargo culting from earlier code related to wire and register symbols (which I've used other places).

Reply:
Alright, this is a wont-fix for now.  This is bumping up against our problems with symbol tables.  This _can't_ be inserted into the symbol table because it crosses an op boundary (FModuleOp/FExtModuleOp) even though that op doesn't have a symbol table.  So, this is going to dangle and we hope it doesn't conflict. 😰 

What I can do is generate a more unique name that includes the module symbol name: `__ModuleName_InterfaceName__`.  Update to use this in the later commits.  

This is something that needs to be discussed with MLIR folks to sort out, but it's just hard to deal with currently.

Reply:
Yeah totally agreed, this shouldn't be a blocker. Just some long term thing that we'll have to deal with in all sorts of places I guess.